roles/magnum/defaults/main.yml

```diff
@@ -44,14 +44,14 @@ magnum_clusterctl_config:
 magnum_image_container_format: bare
 magnum_image_disk_format: raw
 magnum_images:
-  - name: ubuntu-2204-kube-v1.25.11
-    url: https://object-storage.public.mtl1.vexxhost.net/swift/v1/a91f106f55e64246babde7402c21b87a/magnum-capi/ubuntu-2204-kube-v1.25.11.qcow2
+  - name: ubuntu-2204-kube-v1.30.2
+    url: https://static.atmosphere.dev/artifacts/magnum-cluster-api/ubuntu-jammy-kubernetes-1-30-2-1720107688.qcow2
     distro: ubuntu
-  - name: ubuntu-2204-kube-v1.26.6
-    url: https://object-storage.public.mtl1.vexxhost.net/swift/v1/a91f106f55e64246babde7402c21b87a/magnum-capi/ubuntu-2204-kube-v1.26.6.qcow2
+  - name: ubuntu-2204-kube-v1.29.6
+    url: https://static.atmosphere.dev/artifacts/magnum-cluster-api/ubuntu-jammy-kubernetes-1-29-6-1720107687.qcow2
     distro: ubuntu
-  - name: ubuntu-2204-kube-v1.27.3
-    url: https://object-storage.public.mtl1.vexxhost.net/swift/v1/a91f106f55e64246babde7402c21b87a/magnum-capi/ubuntu-2204-kube-v1.27.3.qcow2
+  - name: ubuntu-2204-kube-v1.28.11
+    url: https://static.atmosphere.dev/artifacts/magnum-cluster-api/ubuntu-jammy-kubernetes-1-28-11-1720107685.qcow2
     distro: ubuntu
```

Comment:
Can you actually create in vars.yml a variable called `_magnum_images` with those values, and then set the default of `magnum_images` to `_magnum_images`.

Then, inside the Molecule configuration, instead of having to sync the `magnum_images` all the time, perhaps, you can just make it set to `_magnum_images | last`

Also would be nice in that case to just order them by old to new :)